package.json

```diff
@@ -83,7 +83,7 @@
     "sass": "1.26.9",
     "save-dev": "0.0.1-security",
     "sinon": "9.0.2",
-    "typescript": "3.9.5",
+    "typescript": "4.3.2",
```

Comment:
The npm build just complained about this but it's not a significant upgrade (even though it's a major update).